gum/arch-arm64/gumarm64writer.c

```diff
@@ -120,6 +78,10 @@ static void gum_arm64_writer_put_load_store_pair_pre (GumArm64Writer * self,
 static void gum_arm64_writer_put_load_store_pair_post (GumArm64Writer * self,
     GumArm64MemPairOperandSize op_size, guint opc, gboolean v, gboolean l,
     guint rt, guint rt2, guint rn, gssize post_increment);
+static void gum_arm64_writer_put_load_store_pair_signed_offset (
+    GumArm64Writer * self, GumArm64MemPairOperandSize op_size, guint opc,
+    gboolean v, gboolean l, guint rt, guint rt2, guint rn,
+    gssize signed_offset);
```

Comment:
Is this the same as `gum_arm64_writer_put_load_store_pair_post` except `post_increment` is signed? Because the other one is also meant to be signed, i.e. `post_increment` is `gssize` (signed size_t).